tests/lib/Microsoft.Fx.Portability.MetadataReader.Tests/xunit.runner.json

```diff
@@ -1,4 +1,5 @@
 ﻿{
   "methodDisplay": "method",
-  "diagnosticMessages": true
-}
\ No newline at end of file
+  "diagnosticMessages": true,
+  "shadowCopy": false
```

Comment:
Is necessary so that ilasm will be present?

Reply:
Yup. Since it's never actually referenced by the tests, so it doesn't get copied by xunit :(  I tried adding a comment in there, but it fails to parse with comments